### ISSUE TITLE
Add Unit Tests

### ISSUE THREAD

**Issue by croemmich** [logicalgrape/paypal-ipn-laravel#7]
Something as critical as this needs unit tests!

**Comment by justinsovine** [comment]
Absolutely. After implementing this in a project I'm working on I'll try to go ahead and make a contribution in this light, unless someone else gets to it first.
